.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -1,7 +1,7 @@
 ---
-name: Bug report 🐛
+name: "Bug report \U0001F41B"
```

Comment:
why'd this go away? I miss my 🐛 friend

Reply:
Looks like Github translated bug emoji into unicode. Looks like it's the correct codepoint for bug, tho.